.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+nomad-spark PR Summary
+==============================
+
+Briefly describe how this Pull Request changes the project.
+
+Additional Details
+==============================
+
+If necessary, provide additional context or details about this Pull Request, including all changes not mentioned in the summary.
+
+Checklist
+==============================
+
+Testing
+-------------------------
+
+(Remove this checklist and replace it with "N/A - no code changes" if this PR does not modify source code)
+
+* [ ] I have manually verified that my code changes do the right thing.
+* [ ] I have run all tests and verified that my changes do not introduce any regressions.
+* [ ] I have written unit tests to verify that my code changes do the right thing and to protect my code against regressions.
+* [ ] I have tested my code changes locally or against a cluster using an example Spark project.
+
+Documentation
+-------------------------
+
+(Remove this checklist and replace it with "N/A - no code changes" if this PR does not modify source code)
+
+* [ ] I have updated code comments - both GroovyDoc/JavaDoc-style comments and inline comments - where appropriate.
```

Comment:
Do we need this `both GroovyDoc/JavaDoc-style comments and inline comments` ?

Reply:
Nah, can be removed. Was originally from an Austin Witt template that I cribbed...